app/controllers/ops_controller/settings/common.rb

```diff
@@ -667,7 +667,7 @@ def settings_get_form_vars
           @edit[:new][:server_url] = MiqDatabase.registration_default_values[:registration_server]
         end
       end
-      @changed = (new != @edit[:current])
+      @changed = (new.except(:customer_userid, :customer_password, :customer_verify) != @edit[:current].except(:customer_userid, :customer_password, :customer_verify))
```

Comment:
@mzazrivec why are we excluding these three keys while comparing new/current :-/

Reply:
The idea here was to have the form `Save` button disabled unless credentials validation is successful. So I'm excluding these 3 keys so that the `Save` button remains disabled if you change user login and / or user password (you have to validate the credentials before saving).